Validation/RecoEgamma/python/tkConvValidator_cfi.py

```diff
@@ -10,29 +10,29 @@
 trackAssociatorByHitsForConversionValidation.Cut_RecoToSim = 0.5
 
 from CommonTools.RecoAlgos.trackingParticleRefSelector_cfi import trackingParticleRefSelector as _trackingParticleRefSelector
-tpSelecForEfficiency = _trackingParticleRefSelector.clone()
-tpSelecForEfficiency.chargedOnly = True
+tpSelecForEfficiency = _trackingParticleRefSelector.clone(
+chargedOnly = True,
 # trackingParticleSelector.pdgId = cms.vint32()
```

Comment:
One could have already migrated also the commented out lines... but ok, lets defer it to when they'll get uncommented

Reply:
@perrotta  I would change those commented lines in my next PR. Thanks!